src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -813,7 +802,14 @@ public async ValueTask DisposeAsync()
             private async ValueTask SendIdentitiesAsync()
             {
                 //fire and forget, do not hold-up smuggler operations waiting for Raft command
-                await _database.ServerStore.SendToLeaderAsync(new UpdateClusterIdentityCommand(_database.Name, _identities, false, RaftIdGenerator.NewId()));
+                DevelopmentHelper.ShardingToDo(DevelopmentHelper.TeamMember.Aviv, DevelopmentHelper.Severity.Normal, "try to find a better way to handle restoring identities on sharded database");
```

Comment:
We need to handle the cluster-wide stuff, need to list them all and go over one by one what we want to do..
Please create a sub issue for that.

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-19021/Sharding-Restore-Handle-Compare-Exchange-and-Subscriptions